sites/hashdev/src/_pages/blog/10_error-stack-update-0-2.mdx

```diff
@@ -0,0 +1,185 @@
+
+```rust
+// Note: neither `Debug`, nor `Display` is implemented
+ struct Suggestion(&'static str);
+
+ Report::install_debug_hook::<Suggestion>(|value, context| {
+     context.push_body(format!("suggestion: {}", value.0));
+ });
+
+ let report = read("config.txt").attach(Suggestion("check if `config.txt` exists"));
+```
+
+```
+no such file or directory
+ ├╴read.rs:10:5
+ ╰╴suggestion: check if `config.txt` exists
+```
+
+Report outputs now consist of two sections: a `Body` and an `Appendix`. The `Body` provides top-level information while the `Appendix` includes additional, more verbose details, like a backtrace. Hooks can be used to write to both of these sections.
+
+```
+experiment error: could not run experiment
+ ├╴examples/demo.rs:67:17
+ ├╴backtrace with 19 frames (3)
+ ├╴experiment 3 has no valid description
+ ╰╴unable to set up experiments
+
+ ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+ Backtrace No. 1
+    0: std::backtrace_rs::backtrace::libunwind::trace
+              at /rustc/8b705839cd656d202e920efa8769cbe43a5ee269/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
+    1: std::backtrace_rs::backtrace::trace_unsynchronized
+              at /rustc/8b705839cd656d202e920efa8769cbe43a5ee269/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
+    2: std::backtrace::Backtrace::create
+              at /rustc/8b705839cd656d202e920efa8769cbe43a5ee269/library/std/src/backtrace.rs:333:13
+    3: core::ops::function::FnOnce::call_once
+              at /rustc/8b705839cd656d202e920efa8769cbe43a5ee269/library/core/src/ops/function.rs:251:5
+    ...
+    18: _main
+```
+
+For a full guide to using hooks, please refer to [the `fmt` module documentation](https://docs.rs/error-stack/0.2.0/error_stack/fmt/index.html).
+
+## Related errors and multiple error sources
+
+`error-stack` v0.2 introduces support for related errors and multiple error sources, which can be invaluable when multiple errors occur at the same time, e.g. in loops or in multi-threaded applications. `Report`s are no longer a linear stream of events. You can now create a tree of different `Report`s, enabling the capture of multiple simultaneous errors without loosing information.
+
+This is also shown in the output:
+
+```
+Error: Experiment error: Could not run experiment
+ ├╴examples/demo.rs:73:18
+ ├╴Unable to set up experiments
+ │
+ ├─▶ invalid experiment description
+ │   ├╴examples/demo.rs:43:10
+ │   ╰╴Experiment 2 could not be parsed
+ │
+ ╰┬▶ invalid digit found in string
+  │  ├╴examples/demo.rs:25:18
+  │  ╰╴"3o" could not be parsed as experiment
+  │
+  ╰▶ invalid digit found in string
+     ├╴examples/demo.rs:25:18
+     ╰╴"4a" could not be parsed as experiment
+
+ Experiment error: Could not run experiment
+ ├╴examples/demo.rs:67:17
+ ├╴Experiment 3 has no valid description
+ ╰╴Unable to set up experiments
+```
```

Comment:
Capitalization is wrong here

Reply:
Thanks, fixed in 0e3f7f9ff183c9e0e45df867c8293e81fb1f7419